### PR TITLE
Suggested follows by actor (on profiles) updates

### PR DESCRIPTION
**Pull request by estrattonbailey**
The `isFallback` handling relies on backend https://github.com/bluesky-social/atproto/pull/2805 but this can be deployed independently without negative effects.

Also filtered out followed users from this response. I thought we had done that on the backend, but let's just do it here since backend is swamped right now.

To do:
- [x] Investigate if we need `gcTime` here (seems fine to me)

Compressed the size of this a bit, before/after on desktop web:

CleanShot 2024-09-12 at 10 56 29@2x
CleanShot 2024-09-12 at 10 56 37@2x
CleanShot 2024-09-12 at 11 07 27@2x

Basically no change on mobile, looks like this now:

CleanShot 2024-09-12 at 10 58 13@2x
CleanShot 2024-09-12 at 10 58 19@2x

### CODE REVIEW

File: src/components/FeedInterstitials.tsx [hunk 1]
```diff
@@ -176,9 +173,12 @@ function useExperimentalSuggestedUsersQuery() {
 }
 
 export function SuggestedFollows({feed}: {feed: FeedDescriptor}) {
+  const {currentAccount} = useSession()
   const [feedType, feedUri] = feed.split('|')
   if (feedType === 'author') {
-    return <SuggestedFollowsProfile did={feedUri} />
+    return currentAccount?.did !== feedUri ? (
```
Comment (gaearon) [inline src/components/FeedInterstitials.tsx:179]: nit: this would be a bit easier to read without negation — just following nested structure

```
if (feedType === 'author') {
  if (currentAccount?.did === feedUri) {
    ..
  } else {
    ..
} else {
  ..
}
```

File: src/components/FeedInterstitials.tsx [hunk 2]
```diff
@@ -220,10 +221,12 @@ export function ProfileGrid({
   isSuggestionsLoading,
   error,
   profiles,
+  viewContext = 'feed',
 }: {
   isSuggestionsLoading: boolean
   profiles: AppBskyActorDefs.ProfileViewDetailed[]
   error: Error | null
+  viewContext?: 'profile' | 'feed'
```
Comment (gaearon) [inline src/components/FeedInterstitials.tsx:229]: Let's make this required?

**Review by gaearon — APPROVED** [review]
